src/test/java/com/scottlogic/AuthorPostSorterTest.java

```diff
@@ -0,0 +1,50 @@
+package com.scottlogic;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+import java.util.Arrays;
+import java.util.List;
+
+class AuthorPostSorterTest {
+
+    @Test
+    @DisplayName("Check functionality")
+    void sort_functionality() {
+        AuthorPostSorter authorPostSorter = new AuthorPostSorter();
```

Comment:
Object-oriented (OO) principle: you should work abstractly when possible.
If you only need to use PostSorter methods here, you might want to set the type as `PostSorter authorPostSorter`

Reply:
Ah gotcha, interfaces are relatively new for me but that helps describe their functionality! Fixed

Reply:
The correct term is polymorphism: "the use of a single symbol to represent multiple different types".
Here you have achieve polymorphism through inheritence, not with an interface :)